recon_surf/recon-surf.sh

```diff
@@ -52,8 +52,9 @@ function usage()
     echo "./recon-surf.sh"
     echo -e "\t--sid <subjectID>             Subject ID for directory inside \$SUBJECTS_DIR to be created"
     echo -e "\t--sd  <subjects_dir>          Output directory \$SUBJECTS_DIR (pass via environment or here)"
-    echo -e "\t--t1  <T1_input>              T1 full head input (not bias corrected)"
-    echo -e "\t--seg <segmentation_input>    Name of intermediate DL-based segmentation file (similar to aparc+aseg). Requires an ABSOLUTE Path! Default location: \$SUBJECTS_DIR/\$sid/mri/aparc.DKTatlas+aseg.deep.mgz."
+    echo -e "\t--t1  <T1_input>              T1 full head input (not bias corrected). This must be a conformed image (dimensions: 256x256x256, voxel size: 1x1x1,
+    LIA orientation, and data type UCHAR). Images can be conformed using FastSurferCNN's conform.py script."
+    echo -e "\t--seg <segmentation_input>    Name of intermediate DL-based segmentation file (similar to aparc+aseg). This must be conformed (dimensions: 256x256x256, voxel size: 1x1x1, LIA orientation). FastSurferCNN's segmentations are conformed by default; please ensure that segmentations produced otherwise are conformed. Requires an ABSOLUTE Path! Default location: \$SUBJECTS_DIR/\$sid/mri/aparc.DKTatlas+aseg.deep.mgz."
```

Comment:
I think it would be good to have the command here (python3 conform.py --input <non-conformed-segmentation-input> --output <segmentation-input> --seg_input)

Reply:
In this case, do you mean for `--seg`? If so, I think conform.py currently doesn't support conforming segmentations?